src/esn.calendar.libs/i18n/vi.json

```
@@ -190,6 +190,7 @@
   "Notification: {{summary}}": "Thông báo: {{summary}}",
   "occurrences": "lần lặp",
   "OK": "OK",
+  "On": "trên",
```

Actually in this context this has the wrong meaning. It should be `vào ngày`.

---

And in the English version, the first letter is uppercase, so same for vi:
```suggestion
  "On": "Vào",
```